src/test/java/edu/cornell/kfs/coa/businessobject/GlobalObjectWithIndirectCostRecoveryAccountsTest.java

```diff
@@ -0,0 +1,60 @@
+package edu.cornell.kfs.coa.businessobject;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.kuali.kfs.coa.businessobject.AccountGlobal;
+import org.kuali.kfs.coa.businessobject.AccountGlobalDetail;
+import org.kuali.kfs.coa.businessobject.IndirectCostRecoveryAccount;
+
+import edu.cornell.kfs.coa.fixture.AccountGlobalDetailFixture;
+import edu.cornell.kfs.coa.fixture.AccountGlobalFixture;
+import edu.cornell.kfs.coa.fixture.IndirectCostRecoveryAccountFixture;
+import edu.cornell.kfs.coa.fixture.SubAccountGlobalDetailFixture;
+import edu.cornell.kfs.coa.fixture.SubAccountGlobalFixture;
+import junit.framework.TestCase;
+
+
+public class GlobalObjectWithIndirectCostRecoveryAccountsTest extends TestCase {
```

Comment:
It may be helpful to add some messages to the assertion checks to clarify what the problem is if an assertion fails.